packages/desktop-client/src/components/reports/ReportCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { theme } from '../../style';
+import AnchorLink from '../common/AnchorLink';
+import View from '../common/View';
+
+export default function Card({ flex, to, style, children }) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏should match the filename
```suggestion
export default function ReportCard({ flex, to, style, children }) {
```